src/objects/control.ts

```diff
@@ -84,21 +84,85 @@ export default class Control {
     return new Control(unflatten(flattened));
   }
 
-  toRuby() {
+  // WIP - provides the ability to get the control in its raw form
```

Comment:
Does this comment need updated?

Reply:
I would leave it there for a few deliverables, or until we know we want to keep it or drop it.